src/static/js/modules/show-hide-fields.js

```diff
@@ -0,0 +1,38 @@
+/* ==========================================================================
+ * Show/Hide Additional Fields
+ *
+ * Shows additional fields on forms when additional information is needed
+ *
+ * ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+
+  function toggleAdditionalFields( input ) {
+    var additionalFields = $( input ).attr('data-additional-fields');
+
+    if ( $( input ).is(':checked') ) {
+      $( additionalFields ).show();
+    } else {
+      $( additionalFields ).hide();
+    }
+  }
+
+  // Show or hide additional fields when any radio button in fieldset changes
+  $( '.js-additional-field' ).on('change', function() {
+    var showField = $( this ).find( '.js-additional-field_trigger');
+    toggleAdditionalFields( showField );
+  } );
+
+
+  // Show or hide additional fields on init
+  $( '.js-additional-field' ).each( function ( i ) {
```

Comment:
Will there be situations where we want to show additional fields on init? I'd think no matter what we'd always hide them (unless we start saving form entries to localstorage). If they will always be hidden, we should just do that in the css.